recipes/jupyterlab-spellchecker/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "jupyterlab-spellchecker" %}
+{% set version = "0.5.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/jupyterlab-spellchecker-{{ version }}.tar.gz
+  sha256: c2feb33273810cfd54271a29eaed1ffdb88d3ef0b0a5d33495b5954f164ee269
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
```

Comment:
you may need to specify `--install-option="--skip-npm"`:
https://github.com/jupyter/jupyter-packaging/blob/0.7.12/jupyter_packaging/setupbase.py#L60

Reply:
Seems to work without it :tada: Was it more like "please do it, it is recommended for a reason" or "if it does not work, try this"?

Reply:
the latter: if it's reporting "good" later on, then npm didn't fail silently...